### PR TITLE
header component created, bottom tab navigation with home and word li…

### PR DESCRIPTION
**Pull request by bishownath-d**
homescreen created with header component and bottom tab navigation

### CODE REVIEW

File: src/common/mainStyles.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { StyleSheet } from 'react-native'
+
+export default styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        backgroundColor: '#fff',
+        // alignItems: 'center',
```
Comment (susomshrestha) [inline src/common/mainStyles.js:7]: remove all commented code if not needed

File: src/components/Home/styles.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    cardLabels: {
+        color: 'white',
+        marginTop: 10,
+        fontFamily: 'sans-serif',
+        fontSize: 18
+    },
+    image: {
+        marginTop: 60,
+        // marginRight: -20,
+        marginLeft: -25,
```
Comment (susomshrestha) [inline src/components/Home/styles.js:33]: Is this needed? Why is it a negative margin?

Comment (bishownath-d) [reply src/components/Home/styles.js:33]: negative margin is to make the image look aligned in the card. I will try to resolve it using another way and then push it again.

File: src/common/mainStyles.js (hunk 1 above)
Comment (bishownath-d) [reply src/common/mainStyles.js:7]: Removed 👍

**Review by susomshrestha — APPROVED** [review]

**Pull request merged**